recipes/xtensor/all/test_package/CMakeLists.txt

```diff
@@ -7,11 +7,15 @@ find_package(xtensor REQUIRED CONFIG)
 
 add_executable(${PROJECT_NAME} test_package.cpp)
 target_link_libraries(${PROJECT_NAME} PRIVATE xtensor)
-target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_14)
+target_compile_features(${PROJECT_NAME} PRIVATE cxx_std_17)
```

Comment:
```suggestion
```

Conan 2.x will have settings.compiler.cppstd always, so the C++ standard passed to test package will be passed automatically. This target_compile_features is more a Conan 1.x feature, because in Conan 1.x we didn't have cppstd, so we needed to enforce the C++ standard in the test package.

Reply:
115409b5fca88b76d143038dee3d160a61bc4e3d